tests/test_spark.py

```diff
@@ -156,3 +158,21 @@ def test_get_pings_none_filter(test_store, dummy_pool_executor,
     pings = get_pings(spark_context, app='*')
 
     assert sorted(pings.collect()) == ['value1', 'value2']
+
+
+@pytest.mark.skip("monkeypatching doesn't persist across spark context")
+def test_get_heka_pings(data_dir, test_store, dummy_pool_executor, monkeypatch,
+                        spark_context):
+    from moztelemetry.heka.message_parser import _parse_json
+    mock_parse_json = MagicMock(name='_parse_json',
+                                wraps=_parse_json)
+    import moztelemetry.heka.message_parser
+    monkeypatch.setattr(moztelemetry.heka.message_parser, '_parse_json', mock_parse_json)
+
+    heka_content = open("{}/test_telemetry_gzip.heka".format(data_dir), "rb").read()
```

Comment:
it seemed to work when I enabled the test, although of course the thing I was trying to test didn't. my preference would be just to leave this as-is for now; we can revisit if/when we decide to actually fix the test.